runtime/syntax/cpp.yaml

```diff
@@ -47,9 +47,9 @@ rules:
     - constant.string:
         start: "'"
         end: "'"
-        skip: "\\\\."
+        skip: "(\\\\.)|(.*[^']$)"
```

Comment:
This almost doesn't make sense to me, and only works if the number is at the end of a line. Why `$`?

Reply:
I will reorder the commit. That's outdated change